opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -500,6 +514,36 @@ def test_create_histogram(self):
         self.assertIsInstance(histogram, Histogram)
         self.assertEqual(histogram.name, "name")
 
+    def test_create_histogram_with_advisory(self):
+        histogram = self.meter.create_histogram(
+            "name",
+            unit="unit",
+            description="description",
+            explicit_bucket_boundaries_advisory=[0.0, 1.0, 2],
+        )
+
+        self.assertIsInstance(histogram, Histogram)
+        self.assertEqual(histogram.name, "name")
+        self.assertEqual(
+            histogram._advisory.explicit_bucket_boundaries,
+            [0.0, 1.0, 2],
+        )
+
+    def test_create_histogram_advisory_validation(self):
+        advisories = [
+            {"explicit_bucket_boundaries_advisory": []},
```

Comment:
Should this one be invalid? https://opentelemetry.io/docs/specs/otel/metrics/data-model/#histogram-bucket-inclusivity:~:text=Bucket%20counts%20are%20optional.%20A%20Histogram%20without%20buckets%20conveys%20a%20population%20in%20terms%20of%20only%20the%20sum%20and%20count%2C%20and%20may%20be%20interpreted%20as%20a%20histogram%20with%20single%20bucket%20covering%20(%2DInf%2C%20%2BInf).

Feel free to file as an issue and move on if this is an outstanding issue with the SDK + Views also.

Reply:
Added a test in 3826348ea784396d3eb57590dcd91b0eee7398a4 and appear to work :)